backend/layers/api/portal_api.py

```diff
@@ -48,7 +49,7 @@
 )
 
 
-class PortalApi:
+class PortalApi(ApiCommon):
```

Comment:
Ignore this for now

Reply:
I added this function https://github.com/chanzuckerberg/single-cell-data-portal/blob/9e0a2cb7ecc9b009da4567670fc118cc4355fd7c/backend/portal/api/curation/v1/curation/collections/common.py#L287-L298
We can punt on the double look-up decision and be inefficient for now for the sake of time.